core/src/main/java/com/linecorp/armeria/common/logging/RequestOnlyLog.java

```diff
@@ -271,6 +271,12 @@ default long requestDurationNanos() {
      */
     HttpHeaders requestTrailers();
 
+    /**
+     * Returns the current attempt number of the {@link Request} if the corresponding context is created in
+     * a retrying client. Otherwise, it returns {@code -1}.
```

Comment:
What do you think about clarifying the following:

- Should we return 0, 1 or 2 for the first retry?
- Should we return -1, 0 or 1 for the very first request?

I think it make more sense that:

- Return `0` for the very first request.
- Return `1` for the very first retry.
- Return `2`, `3`, ... for the second retry and so on.